src/uiSelectController.js

```diff
@@ -198,12 +200,29 @@ uis.controller('uiSelectCtrl',
       ctrl.groups.forEach(function(group) {
         ctrl.items = ctrl.items.concat(group.items);
       });
+      addNullItem();
     }
 
     function setPlainItems(items) {
       ctrl.items = items || [];
+      addNullItem();
     }
 
+    function addNullItem() {
+      var nullItem = ctrl.items.some(function(item) {
+        return angular.isArray(item) ? item.some(findNull) : findNull(item);
+      });
+
+      if (!nullItem && ctrl.items.length) {
```

Comment:
We should still add the null item when there are no other items, since this is the only way to clear existing model value

Reply:
Note that when there are no items, it will nows show `(none)` as the only selectable item.

Tbh, it seems like a better behavior than a 2px list / doesn't show anything that currently happens.